lib/near.js

```diff
@@ -31,7 +31,7 @@ class Near {
         this.connection = connection_1.Connection.fromConfig({
             networkId: config.networkId,
             provider: { type: 'JsonRpcProvider', args: { url: config.nodeUrl } },
-            signer: config.signer || { type: 'InMemorySigner', keyStore: config.keyStore || config.deps.keyStore }
+            signer: config.signer || { type: 'InMemorySigner', keyStore: config.keyStore || config.deps?.keyStore }
```

Comment:
@zurgl please, add changes to the `.ts` files. Then run `yarn build`. `near.js` is a build artifact.
Also, can you add a test that is failing without this fix? Thanks!